### PR TITLE
cpprestsdk 2.7.0 (new formula)

### PR DESCRIPTION
**Pull request by ohmyarch**

### CODE REVIEW

**Comment by dunn** [comment]
We'll need a test; [`tinyxml.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/tinyxml.rb) is a good example of a library test. Thanks!

File: Library/Formula/cpprestsdk.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  desc "C++ libraries for cloud-based client-server communication"
+  homepage "https://github.com/Microsoft/cpprestsdk"
+  url "https://github.com/Microsoft/cpprestsdk/archive/v2.7.0.tar.gz"
+  sha256 "d53593fda17bcac8f68af8bc3ba4ac638ea7e7480f43aa7f3350f6f200b0c63e"
+  depends_on "boost"
+  depends_on "openssl"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-DBUILD_TESTS=OFF", "-DBUILD_SAMPLES=OFF", "-DCMAKE_BUILD_TYPE=Release", "Release", *std_cmake_args
```
Comment (dunn) [inline Library/Formula/cpprestsdk.rb]: `DCMAKE_BUILD_TYPE` is already included in `std_cmake_args`: https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/formula.rb#L1042-L1052

File: Library/Formula/cpprestsdk.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    system "getconf _NPROCESSORS_ONLN | make -j"
```
Comment (dunn) [inline Library/Formula/cpprestsdk.rb]: Should be able to do `system "make", "-j#{ENV.make_jobs}"` here instead.

Comment (mikemcquaid) [inline Library/Formula/cpprestsdk.rb]: Pretty sure that happens automatically even without `-j` due to our settings.

**Comment by dunn** [comment]
If you want to trigger a new build, just let us know instead of closing and opening the PR.

**Pull request closed without merging**